roman_numeral_converter_spec.rb

```diff
@@ -0,0 +1,70 @@
+require_relative "./roman_numeral_converter.rb"
+
+describe "Roman Numeral Converter" do
+  context "roman numeral to arabic conversion" do
+    before { @converter = RomanArabicConverter.new }
+
+    context "roman numeral is valid" do
+      context "roman numeral is one character" do
+        specify { expect(@converter.convert_to_arabic("I")).to eq(1) }
+        specify { expect(@converter.convert_to_arabic("V")).to eq(5) }
+        specify { expect(@converter.convert_to_arabic("X")).to eq(10) }
+        specify { expect(@converter.convert_to_arabic("L")).to eq(50) }
+        specify { expect(@converter.convert_to_arabic("C")).to eq(100) }
+        specify { expect(@converter.convert_to_arabic("D")).to eq(500) }
+        specify { expect(@converter.convert_to_arabic("M")).to eq(1000) }
+      end
+      context "addition between roman numerals" do
+        specify { expect(@converter.convert_to_arabic("CXI")).to eq(111) }
+        specify { expect(@converter.convert_to_arabic("CCC")).to eq(300) }
+        specify { expect(@converter.convert_to_arabic("DIX")).to eq(509) }
+      end
+      context "roman numerals that are two characters long" do
+        specify { expect(@converter.convert_to_arabic("IV")).to eq(4) }
+        specify { expect(@converter.convert_to_arabic("IX")).to eq(9) }
+        specify { expect(@converter.convert_to_arabic("XL")).to eq(40) }
+        specify { expect(@converter.convert_to_arabic("XC")).to eq(90) }
+        specify { expect(@converter.convert_to_arabic("CD")).to eq(400) }
+        specify { expect(@converter.convert_to_arabic("CM")).to eq(900) }
+      end
+      context "roman numerals that are complex" do
+        specify { expect(@converter.convert_to_arabic("MCMXLIV")).to eq(1944)}
+        specify { expect(@converter.convert_to_arabic("MMMMCCCXXI")).to eq(4321)}
+      end
+    end
+
+    context "roman numeral is invalid" do
+      specify { expect{ @converter.convert_to_arabic(123) }.to raise_error("input must be a string") }
+      specify { expect{ @converter.convert_to_arabic("XIQ") }.to raise_error("invalid roman input") }
```

Comment:
What about 'xiv' (_lower case_)? Wouldn't this fail with the message 'input must be a string'?